ils_middleware/tasks/amazon/sqs.py

```diff
@@ -35,6 +35,15 @@ def get_group(resource_uri: str) -> str:
     return f"{resource_uri} returned error {result.status_code}"
 
 
+def get_edit_groups(resource_uris: list) -> list:
+    """Retrieves the Resource's editGroups."""
+    edit_groups = []
+    for resource_uri in resource_uris:
+        edit_groups.append(get_group(resource_uri))
+
+    return edit_groups
```

Comment:
I wonder if it might be more pythonic to do this as a list comprehension (a map operation) one-liner:

```python
return [get_group(resource_uri) for resource_uri in resource_uris]
```